watertap/unit_models/pressure_exchanger.py

```diff
@@ -448,16 +601,26 @@ def propogate_state(sb1, sb2):
             self.low_pressure_side.properties_in[0],
             self.low_pressure_side.properties_out[0],
         )
-        self.low_pressure_side.properties_out[
-            0
-        ].pressure = self.low_pressure_side.properties_in[
-            0
-        ].pressure.value + self.efficiency_pressure_exchanger[
-            0
-        ].value * (
-            self.high_pressure_side.properties_in[0].pressure.value
-            - self.low_pressure_side.properties_in[0].pressure.value
-        )
+        if self.config.pressure_exchange_calculation is PressureExchangeType.efficiency:
+            self.low_pressure_side.properties_out[
+                0
+            ].pressure = self.low_pressure_side.properties_in[
+                0
+            ].pressure.value + self.efficiency_pressure_exchanger[
+                0
+            ].value * (
+                self.high_pressure_side.properties_in[0].pressure.value
+                - self.low_pressure_side.properties_in[0].pressure.value
+            )
```

Comment:
I think if you put parentheses around this, black won't make the formatting look so bad